app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -28,7 +31,9 @@ public SessionsListAdapter(List<Session> sessions) {
         Viewholder viewholder = new Viewholder(view);
         return viewholder;
     }
-
+    public static interface AdapterOnClickHandler {
+        void onClick(String name, Viewholder vh);
```

Comment:
why pass the name? shouldn't you be passing the position? then you can get the item from the `adapter.get()` method